### ISSUE TITLE
[Back to Top]: QA: Activation of "back to top" button does not trigger the focus change in Jaws_FF

### ISSUE THREAD

**Issue by gopinathan-n** [carbon-design-system/carbon-for-ibm-dotcom#9705]
### Description

Page Element:
Activation of back to top button

Issue Description:
In Jaws with FF, Activation of "back to top" button does not trigger the focus change. The screen reader focus remains on the button. Note: Working fine in iPhone Safari VO 

Recommend Fix:
After activation of the back to top button, SR focus is expected to be set on top of the page.

Tool:
JAWS_Desktop

### Component(s) impacted

Back to Top

### Browser

Firefox

### Carbon for IBM.com version

v1.40.0-rc.0

### Severity

Severity 4 = The problem is not visible to or noticeable to an average user. Affects minor functionality, no workaround needed.

### Application/website

Carbon for IBM.com

### Package

@carbon/ibmdotcom-web-components

### CodeSandbox example

https://carbon-design-system.github.io/carbon-for-ibm-dotcom/next/web-components https://carbon-design-system.github.io/carbon-for-ibm-dotcom/next/web-components-react

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/carbon-design-system/carbon-for-ibm-dotcom/blob/main/.github/CODE_OF_CONDUCT.md)
- [ ] I checked the [current issues](https://github.com/carbon-design-system/carbon-for-ibm-dotcom/issues) for duplicate issues
